todoapp/todos/serializers.py

```diff
@@ -1,7 +1,137 @@
 from django.contrib.auth import get_user_model
 from rest_framework import serializers
+from .models import Todo
+from users.models import CustomUser
+from projects.models import *
+from django.db.models import Count, Q
+import json
 
+class TaskSerializer(serializers.ModelSerializer):
+    class Meta:
+        model=Todo
+        fields='__all__'
 
-# Add your serializer(s) here
+class User2Serializer(serializers.ModelSerializer):
+    class Meta:
+        model=CustomUser
+        fields=['first_name','last_name','email']
 
+class Task2Serializer(serializers.ModelSerializer):
+    creator=User2Serializer(source='user',read_only=True)
+    created_at = serializers.DateTimeField(source='date_created', format='%I:%M %p, %d %b, %Y')
+    status=serializers.ReadOnlyField()
```

Comment:
```suggestion
    status = serializers.ReadOnlyField()
```
Fix this in all other places as well